app/invocation/invocation_target_group_card.tsx

```diff
@@ -229,6 +229,13 @@ export default class TargetGroupCard extends React.Component<TargetGroupCardProp
                         <ChevronRight className="icon" />
                       </span>
                       <span className="target-label">{target.metadata?.label}</span>{" "}
+                      {target.testSummary && target.testSummary.totalNumCached > 0 && (
+                        <span className="target-cache-status">
+                          {target.testSummary.totalNumCached === target.testSummary.totalRunCount
+                            ? "Cached"
```

Comment:
How about `(cached)` to mimic Bazel's output?

Reply:
The current CSS already added a background + radius to make it a round "badge". I don't think we need the brackets. Im ok with the lower case though if you think it's better visually.